native/explorer/src/encoding.rs

```diff
@@ -248,7 +234,7 @@ macro_rules! unsafe_encode_datetime {
                         $calendar_iso_module,
                         dt_tz.day().encode($env).as_c_arg(),
                         dt_tz.hour().encode($env).as_c_arg(),
-                        (limited_ms, 6).encode($env).as_c_arg(),
+                        (limited_microseconds, 6).encode($env).as_c_arg(),
```

Comment:
Since we now have `time_unit` in scope, we could be choosing between:

* `(limited_milliseconds, 3)`
* `(limited_microseconds, 6)`

if we wanted to. Should we?

Reply:
We probably should, the more precision, the better?

Reply:
Yeah. I think I'm gonna do it on a follow-on PR since it breaks a test.

Reply:
https://github.com/elixir-explorer/explorer/pull/1024